src/index.html

```diff
@@ -17,6 +29,82 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a href="#home">
+        <img
+          src="./images/logo.png"
+          alt="moyo logo"
+        />
+      </a>
+      <nav>
+        <ul class="nav_list">
+          <li>
+            <a
+              href="#apple"
+              class="nav-link is-active"
```

Comment:
Avoid using spaces in the `href` attribute of the `<a>` tag. If 'apple' is meant to be an anchor link, it should start with a `#` symbol.